en/lessons/cleaning-data-with-openrefine.md

```diff
@@ -379,7 +379,7 @@ Once your data has been cleaned, you can take the next step and explore
 other exciting features of *OpenRefine*. The user community of
 *OpenRefine* has developed two particularly interesting extensions which
 allow you to link your data to data that has already been published on
-the Web. The [RDF Refine extension][] transforms plaintext keywords into
+the Web. The [RDF Transform extension][] transforms plaintext keywords into
```

Comment:
The RDF Transform is the successor of this extension. It should work with recent versions of OpenRefine.